### PR TITLE
[IMP] - Add multi company management to utm source

### PR DESCRIPTION
**Pull request by sbejaoui**

### CODE REVIEW

File: utm_source_multi_company/models/utm_source.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+
+class UtmSource(models.Model):
+
+    _inherit = 'utm.source'
+
+    company_id = fields.Many2one(
+        'res.company',
+        'Company',
+        default=lambda self: self.env.user.company_id,
```
Comment (sbidoul) [inline utm_source_multi_company/models/utm_source.py]: _company_default_get

**Review by gva-acsone — APPROVED** [review]
Functional test +1

**Review by RobinetDenisAcsone — APPROVED** [review]
LGTM

**Comment by sbejaoui** [comment]
@OCA/core-maintainers , This PR is ready to merge

**Pull request merged**